app/assets/javascripts/guides/new.js

```diff
@@ -594,23 +592,41 @@ openFarmApp.controller('newGuideCtrl', ['$scope', '$http', '$filter',
   $scope.switchToStep = function(step){
     $scope.step = step;
     $location.hash($scope.step);
+    scrollToTop();
   };
 
+  var scrollToTop = function(){
+    $('body').scrollTop(0);
+  }
+
   $scope.nextStep = function(){
     if ($scope.step === 3){
       $scope.hasEditedStages = true;
     }
     $scope.step += 1;
     $location.hash($scope.step);
+    scrollToTop();
   };
 
   $scope.previousStep = function(){
     $scope.step -= 1;
     $location.hash($scope.step);
+    scrollToTop();
   };
 
-  $scope.nextStage = function(index){
-    $scope.editSelectedStage($scope.stages[index]);
+  var transferStageValuesIfNoneExist = function(stage, nextStage) {
+    if (!$scope.guideExists) {
+      nextStage.environment = stage.environment;
+      nextStage.light = stage.light;
+      nextStage.soil = stage.soil;
+    }
+  }
```

Comment:
I'm going to disable this one because hell hound, semicolons aren't that important. 

https://github.com/feross/standard